examples/WORKSPACE

```diff
@@ -64,25 +64,25 @@ cc_library(name = "z", srcs = glob(["*.c"]), hdrs = glob(["*.h"]))
 )
 
 # Demonstrates a vendored Stackage package to bump a version bound.
-http_archive(
-    name = "split",
-    build_file_content = """
-load("@rules_haskell//haskell:cabal.bzl", "haskell_cabal_library")
-load("@stackage//:packages.bzl", "packages")
-haskell_cabal_library(
-    name = "split",
-    version = packages["split"].version,
-    srcs = glob(["**"]),
-    deps = packages["split"].deps,
-    visibility = ["//visibility:public"],
-)
-    """,
-    patch_args = ["-p1"],
-    patches = ["@rules_haskell_examples//:split.patch"],
-    sha256 = "1dcd674f7c5f276f33300f5fd59e49d1ac6fc92ae949fd06a0f6d3e9d9ac1413",
-    strip_prefix = "split-0.2.3.3",
-    urls = ["http://hackage.haskell.org/package/split-0.2.3.3/split-0.2.3.3.tar.gz"],
-)
+# http_archive(
+#     name = "split",
+#     build_file_content = """
+# load("@rules_haskell//haskell:cabal.bzl", "haskell_cabal_library")
+# load("@stackage//:packages.bzl", "packages")
+# haskell_cabal_library(
+#     name = "split",
+#     version = packages["split"].version,
+#     srcs = glob(["**"]),
+#     deps = packages["split"].deps,
+#     visibility = ["//visibility:public"],
+# )
+#     """,
+#     patch_args = ["-p1"],
+#     patches = ["@rules_haskell_examples//:split.patch"],
```

Comment:
Any pointers on how to regenerate this patch would be most welcome :).

Reply:
The following steps should lead to a working patch:
* Find out which version of `split` is contained in the new snapshot.
* Checkout the `split` source repository at the corresponding git revision.
* Increase the version bound on `base` in that version's Cabal file.
* Generate a patch.